app/vader_sentiment.py

```diff
@@ -1,8 +1,19 @@
+from typing import Callable
+
 from vaderSentiment.vaderSentiment import SentimentIntensityAnalyzer
 
-vader = SentimentIntensityAnalyzer()
 
-# compound score of the sentiment
-def vader_score(text: list) -> list:
+class SentimentAnalyzer:
+    analyzer = SentimentIntensityAnalyzer()
+
+    def __call__(self, text: str) -> float:
+        return self.analyzer.polarity_scores(text)["compound"]
+
+
+sa_class = SentimentAnalyzer()
+print(sa_class("Some text"))
+
 
-    return [vader.polarity_scores(t)["compound"] for t in text]
+positive = "👍"
+neutral = "👌"
+negative = "👎"
```

Comment:
I like the emoji's! For neutral, it might be clearer if you use 🤜, as the one currently used is also a sign for "Okay" but that's just style, take it with a grain of salt. The only thing I would change is either remove them or utilize them as I don't see them in use right now.